### ISSUE TITLE
Mouse scroll doesn't work over contacts list

### ISSUE THREAD

**Issue by max-baz** [wireapp/wire-webapp#6106]
## Wire for web version

2019-03-14-11-05

## Browser version

Latest Chromium (73.0.3683.75)
Latest Wire Desktop (3.7.2891)

## Installed browser addons

Doesn't matter since it reproduces in Wire Desktop.

## Operating system

Arch Linux

## What steps will reproduce the problem?

1. Add a bunch of contacts, so many that they don't fit a single screen
2. Move mouse cursor over contacts panel and try to scroll (I'm actually using touchpad, if that matters)

## What is the expected result?

The list is scrolled

## What is the actual result?

Nothing happens

-----

Since I have a touchscreen, I can also scroll with my finger, and that works. There also is a scrollbar, and I can click and drag it with a mouse. But I want to use the touchpad 😛 
It definitely worked in the past, but I didn't catch the time when it broke.

**Comment by lipis** [comment]
Works for me (macOS) with the touchpad and mouse. Are you still have issues with it?

**Comment by max-baz** [comment]
I confirm it works as expected :)
